app/_locales/en/messages.json

```diff
@@ -3106,6 +3106,12 @@
   "notificationsPortfolioV2Title": {
     "message": "MetaMask Portfolio - join millions"
   },
+  "notificationsSimulationsDescriptionOne": {
+    "message": "Now you can see the potential outcome of your transactions before you make them!"
+  },
+  "notificationsSimulationsDescriptionTwo": {
+    "message": "This is just a simulation, so we can’t guarantee the final outcome. You can turn this off anytime in Settings > Security & Privacy. "
```

Comment:
Do we need a line break after `anytime in` to ensure the settings path get it's own line and doesn't wrap?

Reply:
Good call, I've added a new line after `anytime in`